src/main.ts

```diff
@@ -16,15 +16,21 @@
 
 import { ApiPromise } from '@polkadot/api';
 import { WsProvider } from '@polkadot/rpc-provider';
-import type { BlockHash } from '@polkadot/types/interfaces';
-import { isHex } from '@polkadot/util';
 import * as bodyParser from 'body-parser';
-import * as express from 'express';
-import * as core from 'express-serve-static-core';
-import { BadRequest } from 'http-errors';
+import { RequestHandler } from 'express';
 
-import ApiHandler from './ApiHandler';
+import App from './App';
 import Config, { ISidecarConfig } from './config_setup';
+import BalanceController from './controllers/BalanceController';
+import BlocksController from './controllers/BlocksController';
+import ClaimsController from './controllers/ClaimsController';
+import MetadataController from './controllers/MetadataController';
+import StakingController from './controllers/StakingController';
+import StakingInfoController from './controllers/StakingInfoController';
+import TxArtifactsController from './controllers/TxArtifactsController';
+import TxFeeEstimateController from './controllers/TxFeeEstimateController';
+import TxSubmitController from './controllers/TxSubmitController';
+import VestingController from './controllers/VestingController';
```

Comment:
Is there a better way to do this, like create a `./controllers/index.ts` and then `import * from ./controllers`?